code/src/App.js

```diff
@@ -1,9 +1,171 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Summary } from './components/Summary'
+import './app.css'
 
 export const App = () => {
+
+  //useState Hooks 
+  const [question, setQuestion] = useState(1)
+  const [text, setText] = useState("")
+  const [select, setSelect] = useState("")
+  const [radioButton, setRadioButton] = useState()
+  const [checkbox, setCheckbox] = useState()
+  const [range, setRange] = useState(5)
+  const [submitted, setSubmitted] = useState(false)   //False because form is not submitted from start (initial value)
+
+  //Array with values for mapping radio buttons in #3
+  const radioButtons = ["Apple", "Banana", "Minions?"]
+  //Array with values for mapping checkboxes in #4
+  const checkboxes = ["Yes", "No", "The slowest"]
+
+  //Nextbutton only enabled when text input is over 0 chars
+  const enabled = text.length > 0
+
+  //Function to handle next-button/onclick
+  const handleNext = () => {
+    setQuestion(question + 1)
+  }
+
+  //Function to handle next-button/onclick
+  const handleBack = () => {
+    setQuestion(question - 1)
+  }
+
+  //Function to handle submit-button/oncklick
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    setSubmitted(true)
+  }
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+
+    <div className="app">
+      <h1>&#x2605; Quiz time &#x2605;</h1>
+
+      <form onSubmit={handleSubmit}>
+
+        {/* Show form content if form is not submitted */}
+        {!submitted && (
+          <div className="form-questions">
+
+            {question === 1 && (
+              <div className="question">
+                <label>
+                  <h2># Please enter your name to participate *</h2>
+                  <input
+                    type="text"
+                    required
+                    placeholder="Type your name here..."
+                    onChange={(event) => setText(event.target.value)}
+                    value={text}
+                    // Prevent form to submit if hitting enter after typing name
+                    onKeyPress={(event) => { event.key === "Enter" && event.preventDefault() }}
+                  />
+                </label>
+              </div>
+            )}
+
+            {question === 2 && (
+              <div className="question">
+                <label>
+                  <h2>#1: What's the best location?</h2>
+                  <div className="select-main">
+                    <select
+                      onChange={event => setSelect(event.target.value)}
+                      value={select}
+                    >
```

Comment:
Add `required` to the section?

Reply:
Can add that, wanted to limit the required ones while testing over and over again.